libultraship/libultraship/Window.h

```diff
@@ -51,6 +51,8 @@ namespace Ship {
 			const char* GetKeyName(int32_t scancode);
 			int32_t GetLastScancode();
 			void SetLastScancode(int32_t scanCode);
+			void InitializeAudioPlayer(std::string audioBackend);
+			void InitializeWindowManager(std::string gfxBackend);
```

Comment:
`const std::string&` or `std::string_view`

Reply:
wut does it do?

Reply:
Both prevent string copies. `string_view` is the more modern way to do it but we use `const std::string&` everywhere else.